modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -512,7 +512,17 @@
 	</tr>
 	<tr>
 		<td>INPUT_METHOD</td>
-		<td>//input[contains(@class, 'custom-control-input') and contains(@type,'checkbox')]/..//span[contains(text(),'${inputMethod}')]</td>
+		<td>//input[contains(@class, 'custom-control-input') and contains(@type, 'checkbox')]//..//span[contains(text(), '${key_inputMethod}')]</td>
+		<td></td>
+	</tr>
```

Comment:
If you need to fix your path, you should fix this in the commit that added it. Fixing it in another commit is messy.

Reply:
Fixed